x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/find_cases.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { Cookie } from 'tough-cookie';
+import { User } from '@kbn/cases-plugin/common/api';
+import { UserProfile } from '@kbn/security-plugin/common';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+import { findCasesResp, getPostCaseRequest } from '../../../../common/lib/mock';
+import { findCases, createCase, deleteAllCaseItems } from '../../../../common/lib/utils';
+import { secOnlySpacesAll, superUser } from '../../../../common/lib/authentication/users';
+import { suggestUserProfiles, loginUsers } from '../../../../common/lib/user_profiles';
+import { getUserInfo } from '../../../../common/lib/authentication';
+import { createUsersAndRoles, deleteUsersAndRoles } from '../../../../common/lib/authentication';
+import { securitySolutionOnlyAllSpacesRole } from '../../../../common/lib/authentication/roles';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+  describe('find_cases', () => {
```

Comment:
Could you please add a test where we filter reporters by username and profile ID (`reporters: ['u_123...', 'elastic']`).